admin/dt-prayer-campaigns.php

```diff
@@ -241,12 +283,15 @@ public function main_column() {
                                             </select>
                                         </td>
                                     </tr>
+                                    <tr>
+                                        <td> </td>
+                                    </tr>
                                 </tbody>
                             </table>
```

Comment:
Would it make sense to wrap the select porch section in an else statement coming from the if above?

as choosing a porch only makes sense once you have a campaign created...

Also that would fix the 2 differently looking UI elements of the buttons and the porch select being on top of each other.